### PR TITLE
Add Strings to Translation Function Call in _node_event.html.erb 

### LINKED ISSUE #10106: Add Strings to Translation Function Call in _node_event.html.erb
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file   [_node_event.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/dashboard/_node_event.html.erb) and  [_en.yml](https://github.com/publiclab/plots2/blob/main/config/locales/en.yml) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

 [_node_event.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/dashboard/_node_event.html.erb)
```diff
- 18        <a class="btn btn-outline-secondary btn-sm float-right respond rsvp">RSVP</a>
+ 18            <a class="btn btn-outline-secondary btn-sm float-right respond rsvp"><%= translation('dashboard._node_event.rsvp') %></a>

```
 [_en.yml](https://github.com/publiclab/plots2/blob/main/config/locales/en.yml)
```diff
+  325   _node_event: 
+  326       rsvp: "RSVP"
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by chicken-biryani**
Add Strings to Translation Function Call in _node_event.html.erb .

I have modified line 18 in _node_event.html.erb and added two lines to _en.yml as asked in the issue.

Fixes #10106

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Review by Tlazypanda — COMMENTED** [review]
Hey @chicken-biryani can you please indent the `_node_event: `  line at par with `_header` ? Thanks :v:

**Review by Tlazypanda — COMMENTED** [review]
Hey @chicken-biryani can you move the `_node_event` additions below the `tell_story_blog` line since it seems to be cutting the header entries? Thank you :tada:

**Review by Tlazypanda — COMMENTED** [review]
Hey @chicken-biryani hmm... :thinking:  tests still seem to be failing can you move it above` _header`

File: config/locales/en.yml [hunk 1]
```diff
@@ -328,6 +328,8 @@ en:
         ask_question: "Ask a question"
         post_event: "Post an event"
         tell_story_blog: "Tell your story on the blog"
+    _node_event: 
+      rsvp: "RSVP"
       dashboard: "Dashboard"
```
Comment (noi5e) [inline config/locales/en.yml:333]: I agree with @Tlazypanda's suggestion here, it looks like _node_event is being nested inside the `dashboard` object which is not what we want.

@chicken-biryani can you please move these lines right in between these two lines `by: "by"` (315) and `_node_question` (316)? I think that should do it.

Comment (chicken-biryani) [reply config/locales/en.yml:333]: Definitely.

**Review by noi5e — CHANGES_REQUESTED** [review]
Thanks @chicken-biryani! This looks good, I think it's almost ready to merge.

Before that, I noticed that there's this file that looks like it was committed by accident:

Screen Shot 2021-09-07 at 5 03 47 PM

Can you push another commit deleting this file? We don't want it to get merged into `main`.

**Review by noi5e — APPROVED** [review]
Looks great, thanks for hanging in there!

**Pull request merged**